addons/website_profile/controllers/main.py

```diff
@@ -91,14 +92,28 @@ def get_user_profile_avatar(self, user_id, field='avatar_256', width=0, height=0
             field_name=field, width=int(width), height=int(height), crop=crop
         ).get_response()
 
+    def _prepare_url_from(self):
+        url_from = request.httprequest.headers.get('Referer')
+        no_url = {'url_from_label': None, 'url_from': None}
+        if url_from and ((url_from_parsed := urlsplit(url_from)).netloc == urlsplit(request.httprequest.url).netloc):
```

Comment:
Using url_current variable, making line easier to read in my opinion :)
```suggestion
        if url_from and ((url_from_parsed := urlsplit(url_from)).netloc == urlsplit(url_current).netloc):
```